libraries/node-core-library/src/Sort.ts

```diff
@@ -231,4 +231,92 @@ export class Sort {
       set.add(item);
     }
   }
+
+  /**
+   * Sort the keys given in an object
+   *
+   * @example
+   *
+   * ```ts
+   * console.log(Sort.sortKeys({ c: 3, b: 2, a: 1 })); // { a: 1, b: 2, c: 3}
+   * ```
+   */
+  public static sortKeys<T extends Partial<Record<string, unknown>> | unknown[]>(
+    object: T,
+    { deep, compare }: { deep?: boolean; compare?: (x: string, y: string) => number } = {
+      deep: false,
+      compare: Sort.compareByValue
+    }
+  ): T {
```

Comment:
This method returns the sorted object, but it looks like it also sorts the object in-place. Is there a reason to return the sorted object then? If we want to return an object, I would think it should be a clone of the original with the keys sorted, ensuring that the passed in object isn't manipulated

Reply:
It doesn't sort the object in-place, as it always return a clone. I added test to reflect this